libs/components/ag-grid/src/lib/styles/_base.scss

```diff
@@ -169,6 +169,10 @@ $modernDarkThemeCompact: map.merge($modernDarkThemeBase, $modernThemeCompact);
     cursor: initial;
   }
 
+  .ag-layout-auto-height .ag-body .ag-center-cols-viewport {
+    min-height: var(--sky-ag-grid-min-height, 50px);
```

Comment:
it looks like ag-grid already has a minimum height of 150px, should we use that as the default value so existing grids don't change? 

https://www.ag-grid.com/javascript-data-grid/grid-size/#min-height-with-auto-height

Reply:
I don't know why their docs say 150. I see 50px for that min-height in our themes.